cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/validation/admins/AdminsValidator.kt

```diff
@@ -0,0 +1,13 @@
+package com.cognifide.cogboard.config.validation.admins
+
+import com.cognifide.cogboard.config.validation.Validator
+import java.io.File
+
+object AdminsValidator : Validator {
+    override fun validate(config: String): Boolean {
+        if (File("/data/admins.json").exists()) {
```

Comment:
this validation makes no sense now as i will be always created before code reaches this point... Lets validate if at least one admin is defined with password